examples/src/main/java/org/hiero/sdk/examples/AccountCreateWithAndWithoutAliasExample.java

```
@@ -0,0 +1,114 @@
+        AccountInfo info = new AccountInfoQuery().setAccountId(accountId).execute(client);
+
+        System.out.println("Account's key: " + info.key + " is the same as " + ed25519Key.getPublicKey());
+        System.out.println("Initial EVM address: " + evmAddress + " is the same as " + info.contractAccountId);
+    }
+
+    public void createAccountWithoutAlias() throws Exception {
+        PrivateKey privateKey = PrivateKey.generateECDSA();
+
+        AccountCreateTransaction transaction =
+                new AccountCreateTransaction().setKeyWithoutAlias(privateKey).freezeWith(client);
+
+        transaction.sign(privateKey);
+        TransactionResponse response = transaction.execute(client);
+
+        AccountId accountId = response.getReceipt(client).accountId;
+        AccountInfo info = new AccountInfoQuery().setAccountId(accountId).execute(client);
+
+        System.out.println("Account's key: " + info.key + " is the same as " + privateKey.getPublicKey());
+        System.out.println("Account has no alias: " + isZeroAddress(Hex.decode(info.contractAccountId)));
+    }
+
+    private static boolean isZeroAddress(byte[] address) {
+        for (int i = 0; i < 12; i++) {
+            if (address[i] != 0) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static void main(String[] args) {
+        try {
+            AccountCreateWithAndWithoutAliasExample accountCreateWithAndWithoutAliasExample =
+                    new AccountCreateWithAndWithoutAliasExample();
+
+            accountCreateWithAndWithoutAliasExample.createAccountWithAlias();
+            accountCreateWithAndWithoutAliasExample.createAccountWithBothKeys();
+            accountCreateWithAndWithoutAliasExample.createAccountWithoutAlias();
+
+        } catch (Exception e) {
+            System.err.println("Error creating accounts: " + e.getMessage());
+            e.printStackTrace();
+        }
```

You can just throw the exception from the main func, usually we don't handle anything in the examples.